rpc/grpc/api.go

```diff
@@ -16,7 +17,8 @@ func (bapi *broadcastAPI) Ping(ctx context.Context, req *RequestPing) (*Response
 }
 
 func (bapi *broadcastAPI) BroadcastTx(ctx context.Context, req *RequestBroadcastTx) (*ResponseBroadcastTx, error) {
-	res, err := core.BroadcastTxCommit(req.Tx)
+	// XXX: empty context ... no remote addr!
```

Comment:
Isn't grpc supposed to populate it with some stuff?

Reply:
From https://stackoverflow.com/questions/33684570/session-and-remote-ip-address-in-grpc-go it looks like it doesn't.